c7n/policy.py

```diff
@@ -470,6 +471,15 @@ def run(self, event, lambda_context):
                 self.policy._write_file(
                     "action-%s" % action.name, utils.dumps(results))
         return resources
+    def expand_variables(self, policy):
+        """expand any variables in the action to_from/cc_from fields.
```

Comment:
doc string is referencing the notify origin, worth updating to its purpose here.